modules/apps/batch-engine/headless/headless-batch-engine-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,113 @@
+components:
+    schemas:
+        Task:
+            properties:
+                className:
+                    example: com.liferay.headless.delivery.dto.v1_0.BlogPosting
+                    type: string
+                endTime:
+                    example: 1569261245
+                    format: int64
+                    minimum: 0
+                    type: integer
+                errorMessage:
+                    example: File import failed
+                    type: string
+                executeStatus:
+                    example: INITIALIZED
```

Comment:
is this an enum? or it's more free-form

Reply:
It is the enum. I added it to the definition.